application/management/commands/clear.py

```diff
@@ -0,0 +1,43 @@
+from itertools import chain
+import os
+import shutil
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from application.models import (MovieDirectory, Movie, MovieRequest, 
+                NewMovieNotification, WatchlistItem, Subtitle)
+
+
+class Command(BaseCommand):
+    help = "Clear all Cinema data (for testing purpose)"
+
+    def handle(self, *args, **options):
+        all_models = list(chain(MovieDirectory.objects.all(),
+            Movie.objects.all(), NewMovieNotification.objects.all(), 
+            MovieRequest.objects.all(), WatchlistItem.objects.all(),
+            Subtitle.objects.all()))
+
+        print(f"Delete {len(all_models)} models")
+        for model in all_models:
+            model.delete()
+
+        for folder in ['films', 'posters', 'subtitles']:
+            self.delete_folder(folder)
+
+
+    def delete_folder(self, folder_name):
+        print(f"Delete {folder_name} directory...")
```

Comment:
Also I didn't read this before but why aren't you using `shutil.rmtree(folder_name)` directly instead ?

Reply:
Because it deletes the whole parent entirely, which is not what I want. Those folders are `films`, `posters`and `subtitles`that are exposed through `MEDIA` and in git repo with a `.gitkeep`file. The other way would be to delete and recreate them.

Reply:
It would also delete the `.gitkeep` file so I guess this is better this way.